### ISSUE TITLE
Fatpack and distrbute as a curlable utility

### ISSUE THREAD

**Issue by pplu** [pplu/p5-pod-stripper#2]
For using from a Dockerfile, it may be easier to `curl | perl` the utility

**Comment by bit-man** [comment]
You mean something like ` curl https://raw.githubusercontent.com/pplu/p5-pod-stripper/master/script/pod_stripper.pl | perl ` added to `README.md` ?

**Comment by pplu** [comment]
Yep. It's a bit more complicated, since the script depends on the module `Pod::Strip` being installed. There is a technique called fatpacking, where you can include all the code from all the Perl modules in one file (https://metacpan.org/pod/fatpack).

I've played around with fatpacking the script in https://github.com/pplu/p5-pod-stripper/pull/7, and although I've produced a fatpacked version that works, it's with a patched version of App::Fatpacker.

The way the script is being fatpacked is in the Makefile (`make fatpack`). It may be worth trying https://metacpan.org/pod/fatpack-simple to see if the fatpacking can work without patching App::Fatpacker.
